scripts/on-prem-destroy.sh

```diff
@@ -39,14 +39,21 @@ if [[ ! $(kubectl config use-context "$context") ]]; then
 fi
 
 # You have to wait the default pod grace period before you can delete the pvcs
-grace=$(kubectl --namespace default get sts -l component=elasticsearch,role=data -o jsonpath='{..terminationGracePeriodSeconds}')
+GRACE=$(kubectl --namespace default get sts -l component=elasticsearch,role=data -o jsonpath='{..terminationGracePeriodSeconds}')
+PADDING=120
```

Comment:
is this 120 seconds?  I am wonder if another 30 would be good enough.

Reply:
It is definitely worth trying to find the sweet spot. I will update.